build/osx/wxcocoa.xcodeproj/project.pbxproj

```diff
@@ -6924,6 +6928,8 @@
 /* Begin PBXProject section */
 		19367367C9323490BB936F06 /* Project object */ = {
 			isa = PBXProject;
+			attributes = {
```

Comment:
Seemed strange to me too. But that's probably because of saving with a difference Xcode version than what the file was created with.